pkg/crc/machine/stop.go

```diff
@@ -23,6 +23,9 @@ func (client *client) Stop() (state.State, error) {
 			logging.Debugf("%v", err)
 		}
 	}
+	if err := updateKernelArgs(vm); err != nil {
```

Comment:
this assumes a clean stop is performed.
does this change already occur during the initial start flow?

Reply:
@gbraad no, this is only happen once cluster is provisioned and MCO able to reconcile the config. I think having it   part of stop is better than in the start flow.

Reply:
> this assumes a clean stop is performed.
> does this change already occur during the initial start flow?

if user runs `crc stop -f` instead of `crc stop` this works in the force case also